examples/dreambooth/README.md

```diff
@@ -195,6 +195,17 @@ accelerate launch train_dreambooth.py \
   --max_train_steps=800
 ```
 
+### Using DreamBooth for other pipelines than Stable Diffusion
+
+Altdiffusion also support dreambooth now, the runing comman is basically the same as abouve, all you need to do is replace the `MODEL_NAME` like this:
```

Comment:
"runing", "comman", "abouve" are all misspelled fwiw

Reply:
Thanks! feel free to open a PR to fix this :)